### ISSUE TITLE
Hedwig and Hertz development

### ISSUE THREAD

**Issue by DoctorRainer** [utkarshkukreti/purescript-hedwig#8]
Did you abandon this as well as hertz? If their speed is so awesome as you say it would be great to use it in production in place of elm, cuz pux seems dead and halogen overcomplicated, not to mention that they are both pretty slow. Hedwig lacks subscriptions and hertz is archived, for now I just can use no one of them because of these reasons

**Comment by Kurren123** [comment]
@DoctorRyner I agree this would be a great elm replacement. There seems to be a [PR](https://github.com/utkarshkukreti/purescript-hedwig/pull/6) that has been waiting since 2018 that adds signals, which makes me think that this project has no longer actively maintained. Do you think we can gather some interested people together and create a fork/help maintain this repo?

**Comment by DoctorRainer** [comment]
@Kurren123 I already started to develop my own Elm-like UI library on top of react, but I'm not sure it's usable currently
https://github.com/DoctorRyner/gimel
